benchmarks/dynamo/ci_expected_accuracy/inductor_torchbench_dynamic_inference.csv

```diff
@@ -10,7 +10,7 @@ basic_gnn_gcn,pass,6
 basic_gnn_gin,pass,0
 basic_gnn_sage,pass,0
 clip,pass_due_to_skip,0
-cm3leon_generate,pass,6
+cm3leon_generate,pass,8
```

Comment:
This is benign. Please look at this test why this number is increased - https://github.com/pytorch/pytorch/pull/108528/files#r1315372462